projects/fsastorefrontstyles/scss/_fs-theme.scss

```diff
@@ -24,7 +24,8 @@ $fsSelectors: cx-fs-policy-details, cx-fs-claim-details, cx-fs-premium-calendar,
   cx-fs-policies-chart, cx-fs-message-notification, cx-fs-documents-table,
   cx-fs-searchbox, cx-fs-questionnaire-carousel, cx-fs-navigation-ui,
   cx-fs-comparison-table-container, cx-fs-comparison-table-tab,
-  cx-fs-comparison-table-panel, cx-fs-comparison-table-panel-item !default;
+  cx-fs-quote-comparison, cx-fs-comparison-table-panel,
```

Comment:
Is this needed since there aren't any styles attached to this selector?

Reply:
No need.